ros2param/test/fixtures/param_node.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+from rcl_interfaces.msg import ParameterDescriptor
+import rclpy
+from rclpy.node import Node
+
+
+class ParamNode(Node):
+
+    def __init__(self):
+        super().__init__('param_node')
+        self.declare_parameter('bool_param', True, ParameterDescriptor())
+        self.declare_parameter('int_param', 42, ParameterDescriptor())
+        self.declare_parameter('double_param', 1.23, ParameterDescriptor())
+        self.declare_parameter('str_param', 'Hello World', ParameterDescriptor())
+        self.declare_parameter('int_array', [1, 2, 3], ParameterDescriptor())
+        self.declare_parameter('double_array', [1.0, 2.0, 3.0], ParameterDescriptor())
+        self.declare_parameter('bool_array', [True, False, True], ParameterDescriptor())
+        self.declare_parameter('str_array', ['Hello', 'World'], ParameterDescriptor())
+        self.declare_parameter('byte_array', [b'p', b'v'], ParameterDescriptor())
+        self.declare_parameter('parameter_with_no_value', None, ParameterDescriptor())
```

Comment:
@BMarchi hmm, I'm slightly inclined towards not keeping parameter values here and in the test. You could provide parameter value via overrides when you launch the node in the test itself.

Reply:
> and in the test

What do you mean? Since the node is for test only I don't see the problem of having the parameter declaration here

Reply:
IMHO it's better to have all value definitions in a single place instead of duplicating them across files and keeping them in sync. `launch` provides means to override parameters we could use.